CreeDictionary/utils/paradigm_filler.py

```diff
@@ -116,7 +154,10 @@ def fill_paradigm(
             analysis = lookup_strings[i]
             results_for_cell = sorted(results[analysis])
             # TODO: this should actually produce TWO rows!
-            row[col_ind] = " / ".join(results_for_cell)
+            inflection_cell = row[col_ind]
+            assert isinstance(inflection_cell, InflectionCell)
+            inflection_cell.inflection = " / ".join(results_for_cell)
```

Comment:
I know this was there before, but this is unsatisfying... I might need to make a new issue for this. Basically, an entirely new row should be made for any alternatives.